frontend/testing/cypress/src/support/studio.js

```diff
@@ -1,12 +1,11 @@
 /// <reference types="cypress" />
 import '@testing-library/cypress/add-commands';
-import { login } from '../selectors/login';
-import { designer } from '../pageobjects/designer';
+import '@testing-library/cypress/add-commands';
```

Comment:
Is this import duplicated?

Reply:
Good catch, not sure how this happened 😅